plugins/installer/override/override.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension version="1.0" type="plugin" group="installer" method="upgrade">
```

Comment:
I think we have to change the version number in this line, too. For example see https://github.com/joomla/joomla-cms/blob/933901596831b37494db72820fad601e786bf264/plugins/media-action/crop/crop.xml#L2